app/rules/page.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+
+import Footer from "@/components/footer/footer";
+import Navbar from "@/components/navbar";
+import { formatHourTo12Hour, hackathonDateInfo } from "@/lib/dates";
+
+function Rules() {
+  return (
+    <div className="flex flex-col justify-center items-center min-h-screen">
+      <Navbar />
+      <main className="flex flex-1 w-[100vw]">
+        <div className="flex flex-col flex-1 w-full justify-center items-center mt-48 mx-8 sm:mx-20 md:mx-64 max-w-[50rem]">
+          <h2 className="font-TangoSansBold text-6xl text-primary pb-10 text-center">Rules</h2>
+          <div className="flex flex-col flex-1 w-full justify-center items-center mt-10">
+            <h3 className="font-TangoSansBold text-2xl pb-5 text-[#EE5000] text-center">The Hackathon Spirit</h3>
+            <p className="text-center pb-4">
+              The goal of any hackathon is to encourage participants to learn, build, and share.
+            </p>
+            <ul style={{ listStyleType: "circle" }}>
+              <li>
+                <span className="font-bold">Learn</span>
+                <ul style={{ listStyleType: "square" }} className="ml-8">
+                  <li>Learn new things.</li>
+                  <li>Step outside of your comfort zone.</li>
+                  <li>Experiment and make mistakes.</li>
+                  <li>Don’t be afraid to ask for help from fellow hackers, mentors, or organizers</li>
```

Comment:
This bullet point is missing a period.
```suggestion
                  <li>Don’t be afraid to ask for help from fellow hackers, mentors, or organizers.</li>
```